src/components/AppNavigation/Settings.vue

```diff
@@ -272,6 +290,27 @@ export default {
 		nextcloudVersion() {
 			return parseInt(OC.config.version.split('.')[0])
 		},
+		defaultCalendarOptions() {
+			return this.$store.state.calendars.calendars.filter(calendar => !calendar.readOnly)
```

Comment:
We should now allow incoming shared calendars. The backend only allows owned calendars for now to prevent unnecessary complexity (ACLs etc.).

Reply:
Done